cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -39,12 +39,6 @@
     {% if video_url.find( 'enablejsapi=' ) == -1 -%}
         {% set video_url = _buildParam( video_url, 'enablejsapi=1' )  %}
     {% endif -%}
-    {% if video_url.find( 'origin=' ) == -1 -%}
-        {% set video_url = _buildParam(
-             video_url,
-             'origin=' ~ request.scheme ~ '://' ~ request.META['HTTP_HOST']
```

Comment:
I think that instead of deleting this entirely, changing `request.META['HTTP_HOST']` to `request.get_host()` will fix this in at least beta and prod

Reply:
I think you're right that this will work OK with some other configuration changes in some of our environments. I have amended the commit and rebased.